contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -4029,6 +4041,8 @@ get_bbf_pivot_tuplestore(RawStmt *sql,
 		int			j;
 		int			non_pivot_columns;
 		int			result_ncols;
+		/* only COUNT will output 0 when no row is selected */
+		bool		output_zero = !strcmp(funcName, "count");
```

Comment:
Did you consider "COUNT" ?

Reply:
Could you paste the PG doc url for this comment too

Reply:
bbf on the first place(during bison parser) will lower all function name. We can definitely lower the funcName and do a strcmp with "count"